packages/accordion/accordion.scss

```diff
@@ -1,71 +1,56 @@
 @import "~@fremtind/jkl-core/variables/_all.scss";
 @import "~@fremtind/jkl-core/mixins/_all.scss";
 
-$border-color: $svart;
-$border-width: rem(1px);
-
-// chevron
-$chevron-size: rem(20px);
-$chevron-weight: rem(1px);
-$chevron-weight--hover: rem(2px);
+.jkl-accordion-item {
+    $title-padding: $component-spacing--large;
 
-// Padding
-$content-padding-left-right: $component-spacing--large;
-$content-padding-bottom: $layout-spacing--xl;
-$content-padding-top: $content-padding-bottom - $component-spacing--large;
-$item-padding-top-bottom: rem(2px);
-$title-padding: $component-spacing--large;
-$title-padding-top: $title-padding - $item-padding-top-bottom;
-$title-padding-bottom: $title-padding - $item-padding-top-bottom - $border-width;
+    margin-bottom: $component-spacing--small;
 
-.jkl-accordion {
-    // The border-top is set on the accordion rather than the accordion-item to keep all accordion-item an equal height
-    border-top: $border-width solid $border-color;
-    width: 100%;
-}
+    &:last-child {
+        margin-bottom: 0;
+    }
 
-.jkl-accordion-item {
-    border-bottom: $border-width solid $border-color;
-    padding: $item-padding-top-bottom 0;
+    &:active {
+        transform: scale(0.99);
```

Comment:
Maybe add a slight transition here? It feels a little harsh compared to the other changes (expand, color change)

Reply:
Good catch, I see it now. Updated PR with transition. Also added `transition-property` to content wrapper to narrow down the transition scope.